docker-compose.apps.prod.yml

```diff
@@ -16,6 +16,19 @@ services:
         volumes:
            - ./resources:/app/Resources
 
+    easyspeak_emailer:
+        image: bochka123/easyspeak_emailer:latest
```

Comment:
Change to tatihutii/easyspeak_emailer, it may be changed and tania's account is more persistent here)